src/server.h

```diff
@@ -3725,6 +3732,14 @@ void quitCommand(client *c);
 void resetCommand(client *c);
 void failoverCommand(client *c);
 
+/* io_uring.c -- io_uring related operations */
+#include <liburing.h>
```

Comment:
We can't include this on platforms that don't have it.

Reply:
Yes, you are right, maybe we can solve this by `ifeq ($(USE_IO_URING),yes)`. And use USE_IO_URING to protect it.